CONTRIBUTING

```diff
@@ -40,7 +40,7 @@ imported into LXC).
 
 Language bindings for the libraries need to be released under LGPLv2.1+.
 
-Anything else (non-libaries) needs to be Free Software and needs to be
+Anything else (non-libraries) needs to be Free Software and needs to be
```

Comment:
It's possible this is a license file, in which case you probably don't want to change it, or you'd be better served by refreshing the file w/ the latest version